CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,5 @@
 ### Improvements
 
+- Update to pulumi 3.50 dep and the submodule for pulumi to current master (last bump before introducing renovate bot).
```

Comment:
pulumi 3.150.0?

I don't think users know/care about the submodule and renovate. I'm not even sure there's anything meaningful to note about the update to Pulumi 3.150.0. I fixed the merge conflict and force pushed to the branch, and added a commit that simplified the changelog here to just mention Pulumi 3.150.0.